### PR TITLE
Small fixes for GOLDPANDA.

### PR DESCRIPTION
**Pull request by myidpt**
Fix several bugs in the document related to GOLDPANDA.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ X ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: content/en/news/releases/1.8.x/announcing-1.8.6/index.md [hunk 1]
```diff
@@ -19,7 +19,9 @@ This is the final release of 1.8. Please upgrade your Istio installation to a su
 
 ## Security update
 
-The following 2 CVEs are highly related.
+{{< tip >}}
+The first 2 CVEs are highly related.
+{{< /tip >}}
```
[the remark this answers is not in the document]
Comment (ericvn) [reply content/en/news/releases/1.8.x/announcing-1.8.6/index.md:24]: Similar for the other release doc.

**Comment by ericvn** [comment]
And will will want to cherry-pick this back to master after #9735 is merged.

File: content/en/news/releases/1.8.x/announcing-1.8.6/index.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ericvn) [reply content/en/news/releases/1.8.x/announcing-1.8.6/index.md:24]: I will also notice that we have one CVS link using nvd.nist.gov and another using first.org. I'm not sure if we should use one of those over the other.

[the remark this answers is not in the document]
Comment (myidpt) [reply content/en/news/releases/1.8.x/announcing-1.8.6/index.md:24]: Good catches. Thanks. I updated the change.

**Review by ericvn — APPROVED** [review]

**Pull request merged**